xbmc/utils/FileUtils.cpp

```diff
@@ -249,6 +249,97 @@ CDateTime CFileUtils::GetModificationDate(const int& code, const std::string& st
   return dateAdded;
 }
 
+CDateTime CFileUtils::GetCreationDate(const std::string & strFileNameAndPath)
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types